src/app/components/PaymentSummary/index.tsx

```diff
@@ -4,50 +4,45 @@ import { useSettings } from "~/app/context/SettingsContext";
 
 export type Props = {
   amount: string | number;
-  amountAlt?: string;
   description?: string | React.ReactNode;
   fiatAmount: string;
 };
 
-const PaymentSummary: FC<Props> = ({
-  amount,
-  amountAlt,
-  description,
-  fiatAmount,
-}) => {
+const PaymentSummary: FC<Props> = ({ amount, description, fiatAmount }) => {
   const { t: tCommon } = useTranslation("common");
   const { getFormattedSats } = useSettings();
 
   return (
-    <dl className="mb-0">
-      <dt className="text-sm text-gray-500 dark:text-neutral-500">
-        {tCommon("amount")}
-      </dt>
-      <dd className="text-lg text-gray-700 dark:text-neutral-200 flex flex-row justify-between">
-        <div>{getFormattedSats(amount)}</div>
-        {!!fiatAmount && (
-          <span
-            className="text-gray-500 dark:text-neutral-500"
-            data-testid="fiat_amount"
-          >
-            {" "}
-            (~{fiatAmount})
-          </span>
-        )}
-      </dd>
-      {amountAlt && <dd className="text-gray-400">{amountAlt}</dd>}
+    <dl className="space-y-4">
```

Comment:
this have no effect. maybe add this style on div element instead of dl. but spacing is quite large maybe just remove this

Reply:
Without `space-y-4`:
![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/2e688b66-c92d-49fe-b958-03326e4c6fab)

With `space-y-4`:
![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/7eb565dd-d600-4f7f-9ec0-290588eb69b5)

Reply:
Implemented spacing according to the designs, space is needed desperately.